### PR TITLE
Add help links for DCCsi

### PR DESCRIPTION
**Pull request by HogJonny-AMZN**
Signed-off-by: Jonny Galloway <gallowj@amazon.com>

## What does this PR do?
- fixes the azpy.ahred.ui.help_menu module to function, revised to be dcc agnostic
- updates to rename the maya scene exporter
- add help menus to the exporter
- adds url links for dcc tools in the DCCsi Editor boostrap (menu framework)
- moves some dccsi ui functions out of bootstrap to a new module
- a new about dialog

## How was this PR tested?
1. launch editor and validate 'Studio Tools' menu exists
2. validate each menu item performs its function (starts a tool, or links to a readme)

Editor_shrAxCKfom

image

### CODE REVIEW

**Review by shawstar — APPROVED** [review]

**Review by cgalvan — APPROVED** [review]

File: Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Editor/Scripts/about.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+:file: DccScriptingInterface\\editor\\scripts\\about.py
+"""
+# standard imports
+import os
+from pathlib import Path
+import logging as _logging
+from PySide2.QtCore import Qt
+from PySide2.QtGui import QDoubleValidator
+from PySide2.QtWidgets import QCheckBox, QComboBox, QDialog, QFormLayout, QGridLayout, QGroupBox, QLabel, QLineEdit, QPushButton, QVBoxLayout, QWidget#
```
[the remark this answers is not in the document]
Comment (HogJonny-AMZN) [reply Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Editor/Scripts/about.py]: removed

File: Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Editor/Scripts/about.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+        dccsi_maya_readme_url = "https://github.com/o3de/o3de/tree/development/Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Maya/readme.md"
+
+        dccsi_maya_scene_exporter_url = ""
+
+        dccsi_blender_readme_url = ""
+
+        dccsi_blender_scene_exporter_url = ""
+
+        dccsi_wing_readme_url = ""
```
[the remark this answers is not in the document]
Comment (HogJonny-AMZN) [reply Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Editor/Scripts/about.py]: finished implementing the about dialog, these are filled in and used now

File: Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Editor/Scripts/about.py [hunk 3]
```diff
@@ -18,72 +16,80 @@
...
+                                 f"For help getting started, visit the <a href=\"{dccsi_readme_url}/\">DccScriptingInterface</a> documentation<br/>"
 
-        self.help_label = QLabel()
-        self.help_label.setTextFormat(Qt.RichText)
-        self.help_label.setText(self.help_text)
-        self.help_label.setOpenExternalLinks(True)
+                                 f'or come ask a question in the <a href=\"https://discord.com/channels/805939474655346758/842110573625081876\">O3DE #TechnicalArtists Channel</a> on Discord.</a><br/><br/>'
 
-        self.main_layout.addWidget(self.help_label, 0, Qt.AlignCenter)
+                                 f'The DccScriptingInterface Gem was creaded by <a href=\"https://github.com/HogJonny-AMZN/\">HogJonny-AMZN</a> Along with other AWS Design Technologists. Please let me know if you have ideas, enahncement requests, find any bugs, or would like to contribute.<br/><br/>'
```
Comment (cgalvan) [inline Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Editor/Scripts/about.py:81]:
```suggestion
                                 f'The DccScriptingInterface Gem was created by <a href=\"https://github.com/HogJonny-AMZN/\">HogJonny-AMZN</a> Along with other AWS Design Technologists. Please let me know if you have ideas, enhancement requests, find any bugs, or would like to contribute.<br/><br/>'
```

File: Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Maya/Scripts/Python/scene_exporter/readme.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+- and other things
+
+## Using the Scene Exporter
+
+This section describes how to utilize the Scene Export tool.
+
+### Exporting Scene Objects and FBX Models
+
+Do this ...
```
Comment (cgalvan) [inline Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Maya/Scripts/Python/scene_exporter/readme.md:33]: Should this and the section below be marked as `TODO` or omitted for now?

File: Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/azpy/shared/ui/help_menu.py [hunk 5]
```diff
@@ -51,95 +50,103 @@ class HelpMenu():
     Setup a standard Help item in the menubar for PySide2 GUIs
 
     INPUTS:
-    main_window = the class instance of the QMainWindow
-    tool_label = the menu label for the tool's help item
-    tool_help_page = the http:// path to the tool specific help page
+    @param parent_widget: the class instance of the QMainWindow, or QWidget object
+        note: parent_widget must have it;s own menuBar
```
Comment (cgalvan) [inline Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/azpy/shared/ui/help_menu.py:54]:
```suggestion
        note: parent_widget must have it's own menuBar
```

**Review by cgalvan — APPROVED** [review]

File: Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Maya/Scripts/Python/scene_exporter/readme.md (hunk 4 above)
Comment (HogJonny-AMZN) [reply Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Maya/Scripts/Python/scene_exporter/readme.md:33]: yes, Ben is going to write this doc in the next PR

Comment (HogJonny-AMZN) [reply Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Maya/Scripts/Python/scene_exporter/readme.md:33]: LYN-16100

**Pull request merged**